Getting-Started.md

```diff
@@ -75,13 +75,16 @@ Once you've confirmed that your set is passing checks in your local branch, you
 
 2.  [Create a PR](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) to pull your branch into the master.
 
-1.  You may need to resolve merge conflicts.
+3.  You may need to resolve merge conflicts.
```

Comment:
IMO all of these should be `1.`, to take advantage of Markdown's automatic numbering (so we don't have to manually update it for PRs like this).

You can see that Markdown doesn't actually care that this is `1.` in the current version, it still renders it as `3.` in the generated output. The rest of the numbering is also correct in the rendered output, despite being wrong in the current .md source file.